templates/src/csharp/Grpc.Core/Version.csproj.include.template

```diff
@@ -0,0 +1,9 @@
+%YAML 1.2
+--- |
+  <!-- This file is generated -->
+  <Project>
+    <PropertyGroup>
+      <GrpcCsharpVersion>${settings.csharp_version}</GrpcCsharpVersion>
+      <GoogleProtobufVersion>3.2.0</GoogleProtobufVersion>
```

Comment:
System.Interactive.Async is only included in Grpc.Core.csproj and all other projects get the same version by default. For now I only included Google.Protobuf  because it's a bit special - we have protobuf as a submodule in third_party/submodule and upgrading protobuf is a bit more complicated than bumping other dependencies.